tests/lib/rules/no-unused-prop-types.js

```diff
@@ -3215,6 +3148,23 @@ ruleTester.run('no-unused-prop-types', rule, {
         line: 11,
         column: 8
       }]
+    }, { // None of the props are used
+      code: [
+        'var Hello = createReactClass({',
+        '  propTypes: {',
+        '    unused: PropTypes.string',
```

Comment:
The test case specifically had `React.PropTypes`, and `React.createClass` - are these not necessary to trigger the bug?